src/main/java/com/google/alpollo/AnalysisHelper.java

```diff
@@ -0,0 +1,34 @@
+package com.google.alpollo;
+
+import java.io.IOException;
+
+import com.google.api.gax.rpc.FixedHeaderProvider;
+import com.google.cloud.language.v1.Document;
+import com.google.cloud.language.v1.LanguageServiceClient;
+import com.google.cloud.language.v1.LanguageServiceSettings;
+import com.google.cloud.language.v1.Sentiment;
+
+/**
+ * Helper class for cleaner code and easier testing.
```

Comment:
Please be more specific here, explain what this class, its methods do. I think there's no need to tell that it's "cleaner code and easier testing" :)